src/ui/components/Dashboard/RecordingItem/RecordingListItem.js

```diff
@@ -86,7 +86,7 @@ function ItemTitle({ data, editing, editingTitle, setEditingTitle }) {
           </div>
         ) : null}
       </div>
-      <div className="page-url">Created {moment(data.date).fromNow()}</div>
+      <div className="replay-created-at">Created {moment(data.date).fromNow()}</div>
```

Comment:
perhaps we don't need a className, if we're not going to have any styling

Reply:
Yeah we should be able to just take out the classname here. The styling's being handled [here](https://github.com/RecordReplay/devtools/blob/master/src/ui/components/Dashboard/RecordingItem/RecordingListItem.css#L130-L133) without having to refer to a classname.

Reply:
i don't see my comment i posted before hitting resolve.. good point, i'll go ahead and remove it altogether